environments.json

```diff
@@ -601,7 +601,7 @@
     "short": "FF 46",
     "release": "2016-04-26",
     "retire": "2016-06-07",
-    "obsolete": true
+    "obsolete": "very"
```

Comment:
The rule I am using is true after retirement, and "very" one year later.  This lets me remove one version every time I add a new one.